src/mon/OSDMonitor.cc

```diff
@@ -1833,7 +1833,7 @@ bool OSDMonitor::prepare_failure(MonOpRequestRef op)
   if (m->if_osd_failed()) {
     // add a report
     mon->clog->debug() << m->get_target() << " reported failed by "
-          	       << m->get_orig_source_inst() << "\n";
+                       << m->get_orig_source_inst() << "\n";
```

Comment:
in general, we want to avoid changes only involves formatting changes. it renders  the output of "git blame" less useful.